keepalived/include/snmp.h

```diff
@@ -46,9 +46,10 @@ int header_generic(struct variable *, oid *, size_t *, int,
 
 #define SNMP_DEFAULT_NETWORK_SOCKET	"udp:localhost:705"
 
-#define KEEPALIVED_OID 1, 3, 6, 1, 4, 1, 9586, 100, 5
-#define SNMPTRAP_OID 1, 3, 6, 1, 6, 3, 1, 1, 4, 1, 0
+#define KEEPALIVED_OID {1, 3, 6, 1, 4, 1, 9586, 100, 5}
+#define SNMPTRAP_OID {1, 3, 6, 1, 6, 3, 1, 1, 4, 1, 0}
 #define GLOBAL_OID {KEEPALIVED_OID, 1}
+#define VRRP_OID {KEEPALIVED_OID, 2}
```

Comment:
This shouln't be here. VRRP_OID disappeared to a bad edit on my part - see below.